src/string-class.js

```diff
@@ -0,0 +1,197 @@
+'use strict';
+
+const stringManipulations = {
+  /**
+   * hasVowels method test if a string contains vowels
+   * @return {boolean} true if there are vowels and false otherwise
+   */
+  hasVowels() {
+    return /[aeiou]/i.test(this);
+  },
+
+  /**
+   * toupper method converts all characters to their upper case
+   * @return {string}, a string with upper case characters
+   */
+  toUpper() {
+    return this.replace(/[a-z]/g, (found) => {
+      //makes use of ascii code
+      return String.fromCharCode(found.charCodeAt(0) - 32);
+    });
+  },
+
+  /**
+   * toLower method converts all characters to their lower case
+   * @return {string}, a string with lower case characters
+   */
+  toLower() {
+    return this.replace(/[A-Z]/g, (found) => {
+      //makes use of ascii code
+      return String.fromCharCode(found.charCodeAt(0) + 32);
+    });
+  },
+
+  /**
+   * ucFirst method converts the first character to upper case
+   * @return {string}, a string with it's initial character in upper case'
+   */
+  ucFirst() {
+    return this.replace(this.charAt(0), this.charAt(0).toUpper());
+  },
+
+  /**
+   * isQuestion method checks if a string is a question by
+   * checking if the last character is a question mark ?
+   * @return {boolean}; true if the string is a question and false otherwise
+   */
+  isQuestion() {
+    return /\?$/.test(this.trimSpace()) ? true : false;
```

Comment:
__Code Quality - Nitpick__
When you have a boolean expression _i.e an expression that resolves to `true` or `false`_, it is redundant to add extra logic if you need to return `true`|`false`. this could just be:
```javaScript
return /\?$/.test(this.trimSpace());
```